src/pages/ListOfDogs/index.js

```diff
@@ -0,0 +1,75 @@
+import React, { useEffect, useState } from "react";
+import { fetchDogs } from "../../store/listOfDogs/actions";
+import { useDispatch, useSelector } from "react-redux";
+
+import { Container, Row, Button, CardDeck } from "react-bootstrap";
+import { selectlistOfDogs } from "../../store/listOfDogs/selectors";
+import DogDetails from "../../components/DogDetails";
+
+export default function ListOfDogs() {
+  const dispatch = useDispatch();
+  const dogs = useSelector(selectlistOfDogs);
+
+  // dogs.length === 0 ? console.log("empty") : console.log("present");
+
+  useEffect(() => {
+    dispatch(fetchDogs());
+  }, [dispatch]);
+
+  let filteredDogs = [];
+  const [filter, setFilter] = useState("All");
+
+  if (filter !== "All") {
+    filteredDogs = dogs.filter((dog) => {
+      return dog.breed === filter;
+    });
```

Comment:
when doing an arrow function and immediatly returning, you can use the automatic return (with no `{}` brackets)
```
filteredDogs = dogs.filter((dog) => dog.breed === filter)
```